src/assessment_scripts/det_vs_gt.py

```diff
@@ -61,12 +62,15 @@ def file_loader(files):
 
     acc_gdf = gpd.GeoDataFrame() # accumulator
 
-    crs = None # init
+    #crs = None # init
+    crs = epsg_dft
```

Comment:
I recommend to pass `epsg_dft` as a parameter instead of using a global variable (cf. https://stackoverflow.com/questions/19158339/why-are-global-variables-evil).

Reply:
Yes, I thought about it before and prefer to use a global variable because of the importance to have a unique CRS (by ex. for the clip function) over the all project. 

But having it as a parameter is also good for me.